tests/test_response_model.py

```diff
@@ -0,0 +1,25 @@
+from fastapi import FastAPI
+from fastapi.responses import PydanticResponse
+from fastapi.testclient import TestClient
+from pydantic import BaseModel
+
+
+class Model(BaseModel):
+    name: str
+
+
+app = FastAPI()
+
+
+@app.get("/", response_class=Model)
```

Comment:
Awesome, having a `PydanticResponse` definitely sounds interesting and sweet.

I think you mean `response_model` here.

Reply:
or `response_class=PydanticModel`, not sure but let's have both!

Reply:
Thanks, I fixed it now :)